PhiferTristan/readers-client
# Display All Books

Added a fetch call for all books and a BooksList component to display all the books when at /books.

### Steps to Test

1. pull branch and switch to it
2. npm run dev
3. go to your localhost server url with /books added
4. verify all books from API are in the list
